lib/templates/sentry-client.js

```diff
@@ -1,14 +1,16 @@
 import Vue from 'vue'
-import Raven from 'raven-js'
-import RavenVue from 'raven-js/plugins/vue'
+import * as Sentry from '@sentry/browser'
 
 export default function (ctx, inject) {
-  // Inject Raven to the context as $raven
-  ctx.$raven = Raven
-  inject('raven', Raven)
-  
-  Raven
-    .config('<%= options.public_dsn %>', <%= serialize(options.config) %>)
-    .addPlugin(RavenVue, Vue)
-    .install()
+  const opts = Object.assign({}, <%= serialize(options.config) %>, {
+    integrations: (integrations) => {
+      integrations.push(new Sentry.Integrations.Vue({ Vue }))
+      return integrations
+    }
+  })
+  Sentry.init(opts)
+
+  // Inject Sentry to the context as $sentry
+  ctx.$sentry = Sentry
+  inject('$sentry', Sentry)
```

Comment:
The dollar sign `$` in the `inject()` method is not needed because it will be injected by `nuxt` automatically.